### ISSUE TITLE
Remove auxiliary data from json output

### ISSUE THREAD

**Issue by LuukGijben** [chain-stock/suppy#68]
Auxiliary data, meaning data that is derived from the supply chain model, should be removed from the json output of the `SupplyChain().to_json()` method. This information is essentially duplicate information that is computed by `suppy` in the background on run time is therefor not required.

The fields in particular that are currently included in the output that should be removed are,

- predecessors
- llc

**Comment by LuukGijben** [comment]
@AllexVeldman What do you think of adding a keyword argument to create an option to include such implicit data? This way, in case a user would be interested in it for some reason, e.g. debugging, they would have access to it.

**Comment by AllexVeldman** [comment]
> What do you think of adding a keyword argument to create an option to include such implicit data? This way, in case a user would be interested in it for some reason, e.g. debugging, they would have access to it.

I agree on the predecessors, no real need to have that duplicate information.

For now I would like to steer clear of adding modifier keywords just for the in-case-of. If we keep the llc, which is very little overhead, and drop the predecessors the JSON will be smaller but still contain all the info we have on the supply-chain.

**Comment by LuukGijben** [comment]
I would drop the llc as well. The main reason for the json methods is to improve the ability to share and iterate on cases. The computation of the llc is ambiguous and moreover it gets recalculated anyway. As such I would say it can only create confusion and offers no tangible benefit.

I agree with your point of not adding any keyword modifiers.
